### PR TITLE
Export more deployment info in pkgconfig

### PR DESCRIPTION
**Pull request by doudou**

### CODE REVIEW

File: lib/orogen/templates/deployment.pc [hunk 1]
```diff
@@ -8,6 +8,9 @@ project_name=<%= project.name %>
 deffile=${prefix}/share/orogen/<%= File.basename(project.deffile) %>
 typekits=<%= project.name if project.typekit %> <%= deployer.used_typekits.map { |tk| tk.name }.join(" ") %>
 deployed_tasks=<%= deployer.task_activities.map(&:name).sort.join(",") %>
+deployed_tasks2=<%= deployer.task_activities.
```
Comment (g-arjones) [inline lib/orogen/templates/deployment.pc]: I would have called this `deployed_tasks_pair`

**Comment by g-arjones** [comment]
Looks good 👍

File: lib/orogen/templates/deployment.pc (hunk 1 above)
Comment (doudou) [reply lib/orogen/templates/deployment.pc]: Thanks for being my conscience sometimes ... Renamed to `deployed_tasks_with_models`.

**Comment by g-arjones** [comment]
@doudou you forgot to merge this one

**Pull request merged**